test/extensions/resource_monitors/fixed_heap/fixed_heap_monitor_test.cc

```diff
@@ -2,10 +2,14 @@
 
 #include "source/extensions/resource_monitors/fixed_heap/fixed_heap_monitor.h"
 
+#include "test/test_common/test_runtime.h"
+
 #include "absl/types/optional.h"
 #include "gmock/gmock.h"
 #include "gtest/gtest.h"
 
+using testing::Return;
```

Comment:
move to inner most namespace, in this case the anonymous namespace of the test

Reply:
Done